src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,33 @@
 
 class Animal {
   // write your code here
+  health = 100;
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
```

Comment:
If we read the task description. we will see that

> Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores

So we should change `hidden` to the opposite value (e.g. if it's `true` so the method `hide` should change it to the `false`). Now you always change it to `true` only